kernel/redFlags.mli

```diff
@@ -26,6 +26,7 @@ val fFIX : red_kind
 val fCOFIX : red_kind
 val fZETA : red_kind
 val fCONST : Names.Constant.t -> red_kind
+val fPROJ : Names.Projection.t -> red_kind
```

Comment:
This seems like it should be a Repr.t too since it just drops the unfold flag

Reply:
Yep, missed that one.